msg/vehicle_command.msg

```diff
@@ -95,12 +95,13 @@ float32 param1			# Parameter 1, as defined by MAVLink uint32 VEHICLE_CMD enum.
 float32 param2			# Parameter 2, as defined by MAVLink uint32 VEHICLE_CMD enum.
 float32 param3			# Parameter 3, as defined by MAVLink uint32 VEHICLE_CMD enum.
 float32 param4			# Parameter 4, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float64 param5			# Parameter 5, as defined by MAVLink uint32 VEHICLE_CMD enum.
-float64 param6			# Parameter 6, as defined by MAVLink uint32 VEHICLE_CMD enum.
+float32 param5			# Parameter 5, as defined by MAVLink uint32 VEHICLE_CMD enum.
```

Comment:
Ok, I fixed this by doing the division on the right place and I also check all other users of param5 and param6 and no other needs more precision than float.